include/ebpf_program_attach_type_guids.h

```diff
@@ -27,9 +31,35 @@ extern "C"
     __declspec(selectany) ebpf_attach_type_t EBPF_ATTACH_TYPE_BIND = {
         0xb9707e04, 0x8127, 0x4c72, {0x83, 0x3e, 0x05, 0xb1, 0xfb, 0x43, 0x94, 0x96}};
 
+    /** @brief The programs attached to the INETx_CONNECT hook will be invoked for
```

Comment:
What is the expected return value from the hook? Does 0 mean allow or does 0 mean deny?

Reply:
if this is the only conversation, can I address this in the next PR for this feature? That way the approval votes wont get reset and I can merge these changes.